airflow/contrib/operators/spark_submit_operator.py

```diff
@@ -145,7 +145,7 @@ def execute(self, context):
             name=self._name,
             num_executors=self._num_executors,
             application_args=self._application_args,
-            verbose=self._verbose
+            verbose=self._verbose,
```

Comment:
Could you remove the coma here? Sorry, I'm picky, it just make the perceived PR size smaller by removing one "modified" file